src/commands/init/index.ts

```diff
@@ -100,7 +100,13 @@ export const handleInitializeCli = async (cmdInput: InitProjectInput) => {
 
     const { generatedScriptFile } = await initializeCliProject(projectDirectory, protocolModule, chainId, packageManager, deployerId, allowAnalytics)
 
-    await compileProject(projectDirectory, protocolModule)
+    try {
+      await compileProject(projectDirectory, protocolModule)
+    } catch {
+      console.log(
+        chalkWarning('⚠️ Failed to compile the project. Please run ' + chalkInfo('`bunx infinit compile`') + 'to compile the project after initializing.'),
```

Comment:
- the command is `bunx infinit project compile`
- can we check if the user use bunx, npx, or what